README.md

```diff
@@ -362,6 +362,17 @@ The `SKIPPED` result code indicates that a given test was explicitly skipped by
 * taking a very long time to run
 * failing or passing for the incorrect reason
 
+# Editing And Testing The Debugger Visualizer
+
+### Modify The Visualizer
+
+To modify how components are visualized in the debugger edit the file STL\stl\STL.natvis. For more information on how to modify this file check the [natvis documentation][].

```

Comment:
I think we have one too many "STL"s here; the file we're adding is named `stl\STL.natvis`.
```suggestion
To modify how components are visualized in the debugger edit the file stl\STL.natvis. For more information on how to modify this file check the [natvis documentation][].
```